docs/README.md

```diff
@@ -212,6 +213,22 @@ __Note:__ The default option is `never`.
 
 > :warning: Due to a ["won't fix" bug in the Kubernetes API](https://github.com/kubernetes/kubernetes/issues/41743), the LogDNA agent collects events from the entire cluster, including multiple nodes. To prevent duplicate logs when running multiple pods, the LogDNA agent pods defer responsibilty of capturing events to the oldest pod in the cluster. If that pod is down, the next oldest LogDNA agent pod will take over responsibility and continue from where the previous pod left off.
 
+### Configuring regex for redaction and exclusion/inclusion
+
+You can define rules, using **regex** (regular expressions), to control what log data is ingested:
+ * include *only* specific log lines (with LOGDNA_LINE_INCLUSION_REGEX)
+ * exclude specific log lines (with LOGDNA_LINE_EXCLUSION_REGEX)
+ * redact parts of a log line (with LOGDNA_REDACT_REGEX)
+
+For example, you can identify certain types of logs that are noisy and not needed at all, and then write a regex pattern to match those files and preclude them from ingestion. Conversely, you can use regex expressions to match the log lines that you DO want in to include, and ingest only those log lines. Additionally, you can use the environment variable LOGDNA_REDACT_REGEX to remove certain parts of a log line. Any redacted data is replaced with [REDACTED].
+
+To access our library of common regex patterns, refer to [our regex library documentation](REGEX.md).
+
+Notes:
+   * Inclusion rules overwrite exclusion rules
```

Comment:
This is not entirely correct.

For a line to be ingested, it should match all inclusion rules (if any) and not match any exclusion rule.

For example, if we use `"warning"` as inclusion regex and `"debug"` as exclusion regex (as in, "I want only warnings when those are not included in debug messages"), the following will occur:

- "warning about something" -> will match
- "this is a debug message about a warning" -> will not match
- "something" -> will not match (doesn't match inclusion)

If we want to summarize it,  we can say that exclusion rules overwrite inclusion rules.

Reply:
Glad you caught that, thanks. Fixed it.

Reply:
Oh... I wonder if I meant to say "Inclusion rules **override** (not overwrite) exclusion rules." EIther way, I reworded it as you suggested, that's better.